podcasts/DownloadManager+URLSessionDelegate.swift

```diff
@@ -12,20 +12,20 @@ extension DownloadManager: URLSessionDelegate, URLSessionDownloadDelegate {
 
     // make sure to call the completion handler on the main queue, otherwise it will crash
     func urlSessionDidFinishEvents(forBackgroundURLSession session: URLSession) {
-        #if os(watchOS)
-            DispatchQueue.main.async { [weak self] in
-                guard let self = self, let task = self.pendingWatchBackgroundTask else { return }
+#if os(watchOS)
+        DispatchQueue.main.async { [weak self] in
+            guard let self = self, let task = self.pendingWatchBackgroundTask else { return }
 
-                task.setTaskCompletedWithSnapshot(true)
-            }
-        #else
-            DispatchQueue.main.async { [weak self] in
-                guard let strongSelf = self, let appDelegate = strongSelf.appDelegate(), let backgroundHandler = strongSelf.appDelegate()?.backgroundSessionCompletionHandler else { return }
+            task.setTaskCompletedWithSnapshot(true)
+        }
+#else
+        DispatchQueue.main.async { [weak self] in
+            guard let strongSelf = self, let appDelegate = strongSelf.appDelegate(), let backgroundHandler = strongSelf.appDelegate()?.backgroundSessionCompletionHandler else { return }
```

Comment:
Nit: same here, `guard let self`, then it can be omitted in:
```
guard let self,
let appDelegate = appDelegate(),
let backgroundHandler = appDelegate.backgroundSessionCompletionHandler else {
   return
}
```